src/workerd/api/basics.c++

```diff
@@ -820,6 +820,9 @@ void AbortSignal::triggerAbort(
 void AbortSignal::serialize(jsg::Lock& js, jsg::Serializer& serializer) {
   auto& handler = JSG_REQUIRE_NONNULL(serializer.getExternalHandler(), DOMDataCloneError,
       "AbortSignal can only be serialized for RPC.");
+
+  KJ_LOG(WARNING, "An AbortSignal was serialized for RPC");
```

Comment:
Nit: should should probably be `NOSENTRY` and maybe even `LOG_PERIODICALLY`

Reply:
If on the off chance there's already someone using this a bunch we could end up spamming sentry needlessly. These are most often best left as NOSENTRY as long as someone is actively watching for them

Reply:
Made it both PERIODIC and NOSENTRY